include/mesh/mesh_output.h

```diff
@@ -115,6 +133,15 @@ class MeshOutput
                                  const NumericVector<Number> &,
                                  const std::vector<std::string> &);
 
+  /**
+   * This method should be overridden by output formats wanting to
+   * export discontinuous data.
+   */
+  virtual void write_nodal_data_discontinuous (const std::string &,
+                                               const NumericVector<Number> &,
+                                               const std::vector<std::string> &)
```

Comment:
Wait a minute - why do we have this overload?  We're only overloading the std::vector version, and that gives me -Woverloaded-virtual warnings when the NumericVector version is hidden in the derived class.

Reply:
I'm looking into this now; I thought I had checked for warnings before I pushed. I may have forgotten to include the `using MeshOutput<MeshBase>::write_nodal_data_discontinuous` line in the final version.